README.md

```diff
@@ -1,11 +1,24 @@
 # SDK For Verifiable Credentials in Internet Computer
 
-This repository contains a set of libraries to manage verifiable credentials whether you are an issuer or a relying party. 
+This repository contains a set of libraries to manage verifiable credentials whether you are an issuer or a relying party.
 
 ## Libraries
 
 - [Javascript Library](./js-library/README.md). At the moment only a client for relying parties.
 
+## Projects
+
+- [Dummy Relying Party](./dummy-relying-party/README.md). A canister that provides a dummy relying party to request any kind of credentials.
+- [Dummy Issuer](./dummy-issuer/README.md). A canister is a dummy issuer to issue any kind of credentials.
+
+### Test Projects in Mainnet
+
+There is a deployed version in Internet Computer here: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).
+
+You can test it against the dummy issuer deployed here: [https://qdiif-2iaaa-aaaap-ahjaq-cai.icp0.io/](https://qdiif-2iaaa-aaaap-ahjaq-cai.icp0.io/).
+
+You can use the staging Internet Identity: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).
```

Comment:
Would it make sense to set these values as default on the dummy relying party, such that you don't have to fill anything in to try it out?

People visiting the dummy relying party probably don't necessarily know about this readme so it would be nice if it was even more approachable.

Reply:
Good idea! Done!